### PR TITLE
feat: add bionic compatibillity

### PR DESCRIPTION
**Pull request by viceice**
- add bionic as supported distro
- test install all tools for bionic

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -59,8 +59,27 @@ jobs:
...
+  distro:
+    runs-on: ubuntu-latest
+
+    timeout-minutes: 15
+
+    strategy:
+      matrix:
+        distro:
+          [
+            bionic,
```
Comment (rarkins) [inline .github/workflows/build.yaml:78]: Is this bionic-only because it's an additional test to verify the build?

**Review by rarkins — APPROVED** [review]

File: .github/workflows/build.yaml (hunk 1 above)
Comment (viceice) [reply .github/workflows/build.yaml:78]: It's preparation for other distros later maybe for next ubuntu 22.04 or any other

**Comment by viceice** [comment]
Leaving this as draft until i  fixed the binary builders

File: .github/workflows/build.yaml (hunk 1 above)
Comment (rarkins) [reply .github/workflows/build.yaml:78]: I'm thinking:
- Ensure a Dockerfile build with latest tools works on all supported editions
- Only do deep testing (e.g. lockfile generation) with the latest supported edition (e.g. 20.04 now, 22.04 later)

Comment (viceice) [reply .github/workflows/build.yaml:78]: This is now the case, test only installs latest tools for bionic, no lockfile generation. So this only verifies that all tools can be installed and are executable

**Pull request merged**